raddb/mods-available/smtp

```diff
@@ -0,0 +1,197 @@
+#  -*- text -*-
+#
+#
+#  $Id$
+
+#######################################################################
+#
+#  = SMTP Module
+#
+#  The `smtp` module validates a users name and password against an IMAP server.
```

Comment:
I guess this file has been copied from `rlm_imap` and not been fully updated, it has a lot of references to IMAP